x11-misc/polybar/polybar-3.2.1.ebuild

```diff
@@ -65,3 +67,19 @@ src_configure() {
 	)
 	cmake-utils_src_configure
 }
+
+src_install() {
+	cmake-utils_src_install
+
+	# upstreams installs doc/ into a into a hardcoded directory
+	mv "${ED}/usr/share/doc/${PN}"/config "${ED}/usr/share/doc/${P}"
+	rmdir "${ED}/usr/share/doc/${PN}"
```

Comment:
missing `|| die` on both calls, but
instead of moving files around, this should work
`-DCMAKE_INSTALL_DOCDIR="share/doc/${PF}"`

patch those lines to either respect above variable or `sed` it to install into desired location.

https://github.com/jaagr/polybar/blob/134d33d1bb776d607ee89b181f93326f0482d87f/doc/CMakeLists.txt#L90-L92

and it'll be very cool if you submit a patch upstream fixing hardcoded paths.

because currently it creates both directories
```
/usr/share/doc/polybar-3.2.1/README.md.xz
/usr/share/doc/polybar/config.xz
```

so just make it use versioned dir.